adhocteam/Head-Start-TTADP
# [TTAHUB-181] Add complaint to reason list

## Description of change

Added complaint to activity report reason list.

## How to test

Create a new activity report. Verify that complaint appears in the list, that it is alphabetical, and can be added and saved.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-181


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
